git-secrets-scan-action/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Scan with git-secrets'
+description: 'Scan repository with git-secrets'
+inputs:
+  path:
+    description: 'Relative path under $GITHUB_WORKSPACE to place the repository'
```

Comment:
I think the `path` input should be just be relative to the current working directory (I think that's how all of our actions have been behaving).

The current working directory is independent of the `$GITHUB_WORKSPACE` environment variable, right? And I think neither `actions/checkout@v1` or `actions/checkout@v2` modifies the current working directory. That would also avoid dealing with the behavioral difference between `actions/checkout@v1` and `actions/checkout@v2`.